packs/spells/fortissimo-composition.json

```
@@ -11,7 +11,7 @@
         "damage": {},
         "defense": null,
         "description": {
-            "value": "<p>You call upon your muse to greatly increase the benefits you provide with your <em>courageous anthem</em>, <em>rallying anthem</em>, or <em>song of strength</em> composition. If your next action is to cast one of these compositions, attempt a Performance check. The DC is usually the highest Will DC of the composition's targets, but the GM can assign a different DC based on the circumstances. The effect of your composition depends on the result of your check.</p>\n<hr />\n<p><strong>Critical Success</strong> The status bonus from your composition increases to +3.</p>\n<p><strong>Success</strong> The status bonus from your composition increases to +2.</p>\n<p><strong>Failure</strong> Your composition provides only its normal bonus of +1, but you don't spend the Focus Point for casting this spell.</p>"
+            "value": "<p>You call upon your muse to greatly increase the benefits you provide with your <em>courageous anthem</em>, <em>rallying anthem</em>, or <em>song of strength</em> composition. If your next action is to cast one of these compositions, attempt a <span data-pf2-check=\"performance\"  data-against=\"will\">Performance</span> check. The DC is usually the highest Will DC of the composition's targets, but the GM can assign a different DC based on the circumstances. The effect of your composition depends on the result of your check.</p>\n<hr />\n<p><strong>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Fortissimo Composition Critical]{Critical Success}</strong> The status bonus from your composition increases to +3.</p>\n<p><strong>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Fortissimo Composition]{Success}</strong> The status bonus from your composition increases to +2. </p>\n<p><strong>Failure</strong> Your composition provides only its normal bonus of +1, but you don't spend the Focus Point for casting this spell.</p>"
```

```suggestion
            "value": "<p>You call upon your muse to greatly increase the benefits you provide with your <em>courageous anthem</em>, <em>rallying anthem</em>, or <em>song of strength</em> composition. If your next action is to cast one of these compositions, attempt a @Check[performance|against:will] check. The DC is usually the highest Will DC of the composition's targets, but the GM can assign a different DC based on the circumstances. The effect of your composition depends on the result of your check.</p>\n<hr />\n<p><strong>Critical Success</strong> The status bonus from your composition increases to +3.</p>\n<p><strong>Success</strong> The status bonus from your composition increases to +2. </p>\n<p><strong>Failure</strong> Your composition provides only its normal bonus of +1, but you don't spend the Focus Point for casting this spell.</p>"
```
We don't use the old style inlines anymore, and this should not be two effects. It should be a single effect with a ChoiceSet for the outcome.